app/src/main/java/com/sdp/movemeet/backend/serialization/ActivitySerializer.java

```diff
@@ -23,7 +23,7 @@
     // The key used to access the numberParticipant attribute of a serialized Activity
     public static final String NPART_KEY = "numberParticipant";
     // The key used to access the participantsId attribute of a serialized Activity
-    public static final String PARTICIPANTS_KEY = "participantsId";
+    public static final String PARTICIPANTS_KEY = "participantId";
```

Comment:
This is the trick! ⚠️ Everyone! This field is defined this way on Firebase Firestore: "participantId" (and not "participantsId" with an "s"!). But, yes, this is not really logical since we have generally more than one participant in the sport activity. If anyone of you knows how to rename a field in Firestore Firebase, I would be very interested to know how you would proceed! In the meantime, this field will rename as it is now: **participantId** (WITHOUT A "S"!!!).

Reply:
Makes sense. I think we simply have to change how the object is defined in our code. Then, we'll just have to reset the collection.